build_tools/github_actions/runner/config/start.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Starts the GitHub actions runner as a custom service.
```

Comment:
It just starts it in service mode. Let's also note why we don't user runsvc.sh


```suggestion
# Starts the GitHub actions runner in service mode.
# We don't use runsvc.sh here because it's a shell wrapper around a
# nodejs listener that calls the .NET runner program 😱 in startup mode
# and at the exit codes so it can retry if the runner is busy updating.
# We disable updating, so can avoid those layers of cruft.
```